include/CppUTest/UtestMacros.h

```diff
@@ -249,8 +249,10 @@
 #define BITS_EQUAL_TEXT(expected, actual, mask, text)\
   BITS_LOCATION(expected, actual, mask, text, __FILE__, __LINE__)
 
+#define ROBUST_SIZEOF(type) (2*sizeof(type)/sizeof(short)) /* For the Cl2000, sizeof(char) == sizeof(short) == 1 */
```

Comment:
I agree. One possibility would be PlatformSpecificSizeOf(). I don't like that particularly, either, just for one platform that is quickly getting outdated....

It would be better, imo, to do a "hack" just for the Cl2000, something like redefining sizeof() in a forced include, perhaps? Need to dig a little deeper to determine whether that is possible.